types/ClimateMeasurement.json

```diff
@@ -0,0 +1,20 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "/types/ClimateyMeasurementType",
+    "description": "Climate Measurements",
+    "type" : "object",
+   
+    "properties" : {
+        "value" : {
+            "type" : "number"
+        },
+        "unit" : {
+            "description" : "UNCEFACT unit",
+            "type" :"string"
```

Comment:
Do we want to specify an enum for these types?

Reply:
Can't until we decide what they are. At the moment its looking like maybe we just define an enumeration for each one.